python/example_async.py

```diff
@@ -5,23 +5,23 @@
 import momento.aio.simple_cache_client as simple_cache_client
 import momento.errors as errors
 
-_MOMENTO_AUTH_TOKEN = os.getenv("MOMENTO_AUTH_TOKEN")
-_CACHE_NAME = "cache"
+_MOMENTO_AUTH_TOKEN = os.getenv('MOMENTO_AUTH_TOKEN')
```

Comment:
At some point would be good to agree on formatting/linting - I've been using `black` and that uses `"` rather than `'`

Reply:
@rlinehan it would be really great to add an automated linter to the builds for all of our python processes.  I strongly suspect that everyone will be happy with any modern/popular linter as long as we can:

(a) automate the lint check in CI
(b) have an easy CLI command to run to format the code
(c) decent IDE support.

black seems like it meets all of those criteria so IMO if you had time/interest to add it to some of these repos that would be fantastic and we will all just defer to its preferences :)

it would be really good to get `mypy` in CI as well.

Reply:
Yep! Totally agree, just haven't gotten the time yet 😁 . We've got an issue in the python sdk for this, and I think `black` is what several other folks have recommended too https://github.com/momentohq/client-sdk-python/issues/6.

Reply:
i'm actually working on this now, will chat with you soon :)